### PR TITLE
Add flowtype example again

### PR DESCRIPTION
**Pull request by jribeiro**
Issue:

## What I did
Added flowtype example again

## How to test
Run `cra-kitchen-sink` example

### CODE REVIEW

**Comment by jribeiro** [comment]
Please see the attached image proving the example using flowtype is working. No additional libraries installed.
screen shot 2017-08-08 at 20 52 52

```
$ yarn global list
yarn global v0.27.5
warning package.json: No license field
warning No license field
info "babel-cli@6.24.1" has binaries:
   - babel-doctor
   - babel
   - babel-node
   - babel-external-helpers
```

File: examples/cra-kitchen-sink/src/components/FlowTypeButton.js [hunk 1]
```diff
@@ -0,0 +1,24 @@
+// @flow
+import React from 'react';
```
Comment (danielduan) [inline examples/cra-kitchen-sink/src/components/FlowTypeButton.js]: I don't think our local `.eslintrc` has rules for flow so it's probably a good idea to disable it for this file. `/* eslint-disable */`

**Review by danielduan — APPROVED** [review]
Everything else looks good to me.

File: examples/cra-kitchen-sink/src/components/FlowTypeButton.js (hunk 1 above)
Comment (Hypnosphi) [inline examples/cra-kitchen-sink/src/components/FlowTypeButton.js]: or, even better, add a local .eslintrc

Comment (jribeiro) [inline examples/cra-kitchen-sink/src/components/FlowTypeButton.js]: Good point. Let me sort that out

Comment (danielduan) [inline examples/cra-kitchen-sink/src/components/FlowTypeButton.js]: seems a bit much to add rules for flow just to support an example file but either way, I'll take it.

Comment (jribeiro) [inline examples/cra-kitchen-sink/src/components/FlowTypeButton.js]: Just out of curiosity, how do you get the linting error? I think following this setup should be enough btw:
https://github.com/gajus/eslint-plugin-flowtype#shareable-configurations

Comment (danielduan) [inline examples/cra-kitchen-sink/src/components/FlowTypeButton.js]: Had it in Atom with the linter plugin that reads from the nearest `.eslintrc`.

**Comment by ndelangen** [comment]
snapshot needs an update:
```
FAIL  examples/cra-kitchen-sink/src/storyshots.test.js
  ● Console

    console.info node_modules/@storybook/react/dist/server/babel_config.js:73
      => Loading custom .babelrc
    console.error node_modules/fbjs/lib/warning.js:35
      Warning: DocgenButton: type specification of prop `style` is invalid; the type checker function must return `null` or an `Error` but returned a function. You may have forgotten to pass an argument to the type checker creator (arrayOf, instanceOf, objectOf, oneOf, oneOfType, and shape all require an argument).

  ● Storyshots › AddonInfo.DocgenButton › DocgenButton

    expect(value).toMatchSnapshot()
    
    New snapshot was not written. The update flag must be explicitly passed to write a new snapshot.
    
    This is likely because this test is run in a continuous integration (CI) environment in which snapshots are not written by default.
      
      at Object.test (node_modules/@storybook/addon-storyshots/dist/test-bodies.js:27:18)
      at Object.<anonymous> (node_modules/@storybook/addon-storyshots/dist/index.js:155:25)
          at Promise (<anonymous>)
      at Promise.resolve.then.el (node_modules/p-map/index.js:42:16)
          at <anonymous>

  ● Storyshots › AddonInfo.FlowTypeButton › FlowTypeButton

    expect(value).toMatchSnapshot()
    
    New snapshot was not written. The update flag must be explicitly passed to write a new snapshot.
    
    This is likely because this test is run in a continuous integration (CI) environment in which snapshots are not written by default.
      
      at Object.test (node_modules/@storybook/addon-storyshots/dist/test-bodies.js:27:18)
      at Object.<anonymous> (node_modules/@storybook/addon-storyshots/dist/index.js:155:25)
          at Promise (<anonymous>)
      at Promise.resolve.then.el (node_modules/p-map/index.js:42:16)
          at <anonymous>
```

**Comment by ndelangen** [comment]
@danielduan if https://github.com/storybooks/storybook/tree/info-docgen is your branch, it's up to you to merge 👍

**Review by ndelangen — APPROVED** [review]

**Pull request merged**